packages/loot-core/src/types/api-handlers.d.ts

```diff
@@ -63,6 +63,13 @@ export interface ApiHandlers {
     flag: boolean;
   }) => Promise<void>;
 
+  'api/hold-budget-for-next-month': (arg: {
```

Comment:
One thought I had is it might be good to call this `budget-hold-for-next-month` (and use the same pattern for the other endpoint) vs the current `hold-budget-for-next-month`. My thought process is that the former sounds more like you're holding an _amount_ from the budget for next month (the additive nature of the feature), whereas the latter is holding the entire budget. Doing the former also matches previous endpoints in the file (e.g. `budget-set-amount`, `budget-set-carryover`)

Reply:
I agree. Changing it